src/System.Linq.Expressions/tests/CompilerTests.cs

```diff
@@ -27,7 +27,7 @@ public static void CompileDeepTree_NoStackOverflow(bool useInterpreter)
             Assert.Equal(n, f());
         }
 
-        [Theory, ClassData(typeof(CompilationTypes))]
+        [Theory(Skip = "See https://github.com/dotnet/corefx/issues/15533"), ClassData(typeof(CompilationTypes))]
```

Comment:
I think the fact that it worked before means the assertion described at https://github.com/dotnet/coreclr/issues/9158 is bogus and should be taken out, in which case this will work again.

If it's decided otherwise, then this test should indeed be deleted.